mods/beds/functions.lua

```diff
@@ -224,6 +238,37 @@ minetest.register_on_leaveplayer(function(player)
 	end
 end)
 
+minetest.register_on_dieplayer(function(player)
+	local pos = player:get_pos()
+	local yaw = get_look_yaw(pos)
+
+	lay_down(player, nil, pos, false)
```

Comment:
this should only be done if the player actually was in bed

Reply:
Done